src/components/Person/Person.jsx

```diff
@@ -1 +1,25 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+import './Person.scss';
+
+export const Person = ({ person }) => {
+  const { name, age, sex, isMarried, partnerName } = person;
+  const partnerSex = (sex === 'm' && 'wife') || 'husband';
```

Comment:
Wow... this is very strange, let's use the ternary operator

Reply:
Ok 😂